token/services/identity/msp/x509/deserializer.go

```diff
@@ -80,3 +84,51 @@ func (a *AuditInfoDeserializer) DeserializeAuditInfo(raw []byte) (*AuditInfo, er
 	}
 	return ai, nil
 }
+
+func GetEnrollmentID(id []byte) (string, error) {
+	si := &msp.SerializedIdentity{}
+	err := proto.Unmarshal(id, si)
+	if err != nil {
+		return "", errors.Wrap(err, "failed to unmarshal to msp.SerializedIdentity{}")
+	}
+	block, _ := pem.Decode(si.IdBytes)
```

Comment:
We could re-use PemDecodeCert(si.IdBytes) to avoid some duplication

Reply:
👍